src/content/getting-started/quickstart/external/_index.md

```diff
@@ -79,7 +76,7 @@ subctl deploy-broker --kubeconfig kubeconfig.cluster-a --globalnet
 
 When Submariner joins a cluster to the broker via the `subctl join` command, it chooses a node on which to install the
 gateway by labeling it appropriately. By default, Submariner uses a worker node for the gateway; if there are no worker
-nodes, then no gateway is installed unless a node is manually labeled as a gateway. Since we are deploying k3s all-in-one
+nodes, then no gateway is installed unless a node is manually labeled as a gateway. Since we are deploying all-in-one
```

Comment:
I'm not sure the "all-in-one" reference here makes sense since L51-53 above were removed. I think this section exists b/c it assumes K3s was used to deploy the clusters. Since it seems we're not recommending or assuming any specific cluster environment. perhaps we don't need this section.

Reply:
To maintain consistency with the document's flow, it's necessary to have an all-in-one cluster, regardless of whether it's a K3s environment or not.

Reply:
I don't see anywhere else in this doc where it mentions or requires all-in-one clusters plus the section above where it recommends to setup all-in-one clusters, ie K3s, was removed. Perhaps we should still recommend some flavor of an all-in-one cluster w/o specifics? Or assume gateway nodes are properly labelled and remove this section?

Reply:
Single node cluster requirement is mentioned in the prerequisites section - Line 47 and 48

Reply:
@tpantelis @sridhargaddam So as I understand it, there aren't any changes requested?